lib/git-hooks.js

```diff
@@ -3,6 +3,12 @@ var util = require('util');
 var spawn = require('child_process').spawn;
 var fs = require('fs');
 var fsHelpers = require('./fs-helpers');
+var execSync = require('child_process').execSync;
+if (!execSync) {
+    execSync = require('runsync').exec;
```

Comment:
What if we install runsync always?


I just use sync version because this test fails
```
  1) git-hook runner when hooks are found and a hook is unexecutable should return an error:
     AssertionError: expected [Function] to throw Error
```
if you think that async exec will be better than I'll rewrite the code.